plugins/modules/monitoring/uptimerobot.py

```diff
@@ -7,139 +7,183 @@
 __metaclass__ = type
 
 
-DOCUMENTATION = '''
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
 
+DOCUMENTATION = '''
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptimerobot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptimerobot
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
+version_added: "1.9"
 requirements:
     - Valid Uptime Robot API Key
 options:
     state:
         description:
             - Define whether or not the monitor should be running or paused.
         required: true
-        choices: [ "started", "paused" ]
+        choices: [ "started", "paused", "created", "absent", "present" ]
+    url:
+        description:
+            - URL to be checked.
+        required: true
+        version_added: '2.10'
```

Comment:
```suggestion
        version_added: '1.2.0'
```
Same for the others.

Reply:
Fixed in: https://github.com/ansible-collections/community.general/pull/961/commits/12ad8ea0388fe96e3606eb99335cec60233185f3